4-WebApp-your-API/4-3-AnyOrg/TodoListService/ToDoListService.csproj

```diff
@@ -15,8 +15,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Identity.Web" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="1.2.0" />
+    <PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="1.2.0" />
+    <PackageReference Include="Microsoft.Identity.Web.UI" Version="1.2.0" />
```

Comment:
Is Web.UI needed here?

Reply:
Only MicrosoftGraph is needed.
(Web will be drawn by Graph)